.gitattributes

```diff
@@ -9,3 +9,6 @@
 /docs/** linguist-documentation
 /docs/analytics/** -linguist-documentation
 /website/** linguist-documentation
+
+# Auto detect text files and perform LF normalization
+* text=auto
```

Comment:
How does this fix the issue you described in your email? From my understanding of the documentation, this gitattributes entry is equivalent to doing nothing. It's a no-op.

Reply:
Sorry,  this is incomplete and doesn't do what the title says. There are 3 more lines + comments that needs to go with this. I will add them now.